setup.cfg

```diff
@@ -58,6 +58,10 @@ install_requires =
     tornado>=6.1.0  # matches jupyter_server value
     traitlets>=5
 
+    # Fix lack of upper pin for rx in graphql-core<2.3.0 (remove when
+    # upgrading graphene to 3.0):
+    rx<2
```

Comment:
The other option is to pin `graphql-core>=2.3.0`?

Reply:
graphql-core 2.3.2 was the last v2 release, due to graphene-tornado and graphql-ws (both stuck on v2) we cannot move up to v3.

Reply:
The conda-forge feedstock lower-pins the dep at 1.6, should this be done here too?

https://github.com/conda-forge/cylc-uiserver-feedstock/blob/d45ddfd4cb0acf9b4a30a50e6f3294caea16dfc0/recipe/meta.yaml#L48

Reply:
- There is already [a requirement on `graphql-core>=2.0,<3` in `graphene 2.1.0 ` and above](https://github.com/graphql-python/graphene/blob/05d7e61b844b574f59d68ead86ed40431843d51d/setup.py#L90), so raising the lower boundary would give us `graphql-core>=2.3.0,<3`

- There is already [a requirement on `rx>=1.6` in `graphql-core 2.2` and above](https://github.com/graphql-python/graphql-core-legacy/blob/4b1f697ac068a5ff83c35cd818a430e35a0b14b4/setup.py#L24), so we shouldn't need to duplicate the lower boundary necessarily

Reply:
ok